docs/docs/tutorials/get-started/README.mdx

```diff
@@ -113,7 +113,7 @@ ghcr.io/logto-io/logto:prerelease
 :::tip
 
 - If you are using Docker Hub, use `svhd/logto:prerelease` instead of `ghcr.io/logto-io/logto:prerelease`.
-- Use `host.docker.internal` in `DB_URL` to refer to the host IP.
+- Use `host.docker.internal` or `172.17.0.1` in `DB_URL` to refer to the host IP.
```

Comment:
Be better to add platform info?

Reply:
Sorry for the late response. I don't think this is a platform specific setting, as we can also use `host.docker.internal` on linux with some extra configs.
So, better just list the IP here as a fallback workaround. "If the former URL doesn't work for you, try the IP."